client/test/unit/specs/transactions.spec.js

```diff
@@ -166,53 +196,99 @@ describe('Testing Transactions.vue', () => {
+    expect(vm.$el.querySelector('#transaction-list').textContent).toContain(
+      'There was an error loading the transaction history for this NR. Please try again by reloading the page.'
+    )
+  })
+
+  it('displays the transaction header', () => {
+    expect(vm.$el.querySelector('#transaction-header-title').textContent).toContain('NR 1234567')
+    expect(vm.$el.querySelector('#transaction-header-title').textContent).toContain('Priority')
+    expect(vm.$el.querySelector('#transaction-header-title').textContent).toContain('N/A')
+
+    expect(vm.$el.querySelector('#transaction-header-names').textContent).toContain('First')
+    expect(vm.$el.querySelector('#transaction-header-names').textContent).toContain('Looks good')
+    expect(vm.$el.querySelector('#transaction-header-names').textContent).toContain('Second')
+    expect(vm.$el.querySelector('#transaction-header-names').textContent).toContain('Not good')
 
-    expect(data.vm.$el.querySelector('#transaction-list').textContent)
-      .toContain('There was an error loading the transaction history for this NR. Please try again by ' +
-        'reloading the page.');
+    expect(vm.$el.querySelector('#transaction-header-info').textContent).toContain('Submitted Date:')
+    expect(vm.$el.querySelector('.submitted-date').textContent).toBe('2021-07-12, 9:49 am Pacific time')
+
+    expect(vm.$el.querySelector('#transaction-header-info').textContent).toContain('Expiry Date:')
+    expect(vm.$el.querySelector('.expiry-date').textContent).toBe('2021-09-11, 12:01 am Pacific time')
+
+    expect(vm.$el.querySelector('#transaction-header-info').textContent).toContain('Request Status:')
+    expect(vm.$el.querySelector('.request-status').textContent).toContain('APPROVED / Used for BC1234567')
+
+    expect(vm.$el.querySelector('#transaction-header-info').textContent).toContain('Consent:')
+    expect(vm.$el.querySelector('.consent-text').textContent).toBe('Required. Not Yet Received.')
+
+    expect(vm.$el.querySelector('#transaction-header-info').textContent).toContain('Additional Information:')
+    expect(vm.$el.querySelector('.addl-info').textContent).toBe('The quick brown fox...')
   })
 
-  test('It displays default transactions', () => {
-    expect(data.vm.$el.querySelector('#transaction-list').textContent).toContain('Marked on Hold');
-    expect(data.vm.$el.innerHTML).toContain('Marked on Hold')
-    expect(data.vm.$el.innerHTML).toContain('Decision')
-    expect(data.vm.$el.innerHTML).toContain('Created NR (Payment Completed)')
-    expect(data.vm.$el.innerHTML).toContain('Created NR (Unknown)')
-    expect(data.vm.$el.innerHTML).toContain('Cancelled in Name Request')
-    expect(data.vm.$el.innerHTML).toContain('Edit NR Details after Completion')
-    expect(data.vm.$el.innerHTML).toContain('Reset')
+  it('displays a transaction history list item', () => {
+    const item = vm.$el.querySelectorAll('.transaction-item')[0]
+
+    // FUTURE: verify values for all of these...
```

Comment:
I ran out of time. (I spent enough time on this already.)

Reply:
Totally fine, Rome('s unit tests) weren't built in a day.